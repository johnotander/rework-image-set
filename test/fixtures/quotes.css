.bg-img {
  background-image: image-set("my-img.png" 1x);
}
