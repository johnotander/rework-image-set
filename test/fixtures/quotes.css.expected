.bg-img {
  background-image: url('my-img.png');
  background-image: image-set('my-img.png' 1x);
}
